.github/pull_request_template.md

```diff
@@ -1,14 +1,59 @@
-### PR Checklist
+<!-- ⚠️⚠️ Do Not Delete This! pr_template ⚠️⚠️ -->
+<!-- Please read our Rules of Conduct: https://github.com/OHIF/Viewers/blob/master/CODE_OF_CONDUCT.md -->
+<!-- 🕮 Read our guide about our Contributing Guide here https://v3-docs.ohif.org/development/contributing -->
+<!-- :hand: Thank you for starting this amazing contribution! -->
 
-- [ ] Brief description of changes
-- [ ] Links to any relevant issues
-- [ ] Required status checks are passing
-- [ ] User cases if changes impact the user's experience
-- [ ] `@mention` a maintainer to request a review
+### Context
 
 <!--
-  Links
-  -->
+Provide a clear explanation of the reasoning behind this change, such as:
+- A link to the issue being addressed, using the format "Fixes #ISSUE_NUMBER"
+- An image showing the issue or problem being addressed
+- Error logs or callStacks to help with the understanding of the problem
```

Comment:
Is this really necessary? Shouldn't this type of information exist already in the original issue?

Reply:
I don't see why not?

Reply:
done